src/opera_disp_tms/generate_opera_disp_tile.py

```diff
@@ -79,6 +83,9 @@ def main():
     args.begin_date = datetime.strptime(args.begin_date, '%Y%m%d')
     args.end_date = datetime.strptime(args.end_date, '%Y%m%d')
 
+    if args.edc_username and args.edc_password:
```

Comment:
@forrestfwilliams how built out do you think this should be? Should we handle a bunch of exceptions or do a simple check-if-they're-there?